### ISSUE TITLE
Double loading vue with message You are running Vue in development mode. 

### ISSUE THREAD

**Issue by Sebarkar** [fusioncharts/vue-fusioncharts#70]
![2020-08-09](https://user-images.githubusercontent.com/44950137/89730601-bf49b280-da48-11ea-9d3b-aa3ff7399728.png)

`import VueFusionCharts from 'vue-fusioncharts';
import FusionCharts from 'fusioncharts';
import Charts from 'fusioncharts/fusioncharts.charts';
import FusionTheme from 'fusioncharts/themes/fusioncharts.theme.fusion';

Vue.use(VueFusionCharts, FusionCharts, Charts, FusionTheme);`

When I commented this code Vue working normal.

**Comment by sebastian-wobiz** [comment]
have the same error but, it also appears in development mode in the production build.

**Comment by Casal0x** [comment]
Yes i have the same issue, but its happening from fusioncharts on production mode.

**Comment by UZaitsau** [comment]
I have the same issues:
- duplicating of the warning in dev mode
- warning even in production mode

I guess that main reason for that is validation checks in library code like `process.env.NODE_ENV !== 'production'`. 
The `process` object during the building stage provides information about the current Node.js process. But in runtime it belongs to browser.
I'm using webpack as a bundler.
Any thoughts?

**Comment by qkjosh** [comment]
Just wanted to chime in to say I'm seeing this issue as well. It appears that the entire dev mode version of Vue (version 2.5.17) is included in both the `dist/vue-fusioncharts.js` and `dist/vue-fusioncharts.min.js` files. Most people will already be importing Vue with their own bundler or via CDN and don't need this other copy showing up.
